R/utils-get_code_dependency.R

```diff
@@ -54,6 +59,10 @@ get_code_dependency <- function(code, names, check_names = TRUE) {
   as.character(code[unique(c(lib_ind, ind))])
 }
 
+is_symbol <- function(call_pd, symbol) {
```

Comment:
Please add a comment explaining what exactly this function does. I hardly remember what I envisioned two weeks ago myself.

Also, `@internal`.

Reply:
Hey @chlebowa thanks for taking the time before New Years and now to review that!
I added a documentation to `is_symbol` eca80e690dfd3bdc92b08b953aec9024fd3adc9b

There is a lot of text added as a pay-off of the substitution of 3 lines with a new function, but in few months it's gonna be useful to understand what's the purpose of the function, even though it's a one liner function.